modules/ROOT/pages/6.7-release-notes.adoc

```diff
@@ -150,7 +154,13 @@ The {productname} 6.7.0 release includes an accompanying release of the **PowerP
 
 ==== Stopped pasting comments from word documents.
 // TINY-9975
+Previously, pasting content from a word document that contained comments were copied alongside the document's content, resulting in the unintended inclusion of comments when pasting it into a {productname} document.
+
+{productname} 6.7, introduced a solution for xref:introduction-to-powerpaste.adoc[PowerPaste] that now excludes comments when pasting data into a {productname} document.
+
+As a result of this fix, users can now paste content from Word documents without the inconvenience of comments being inadvertently included in the paste operation.
 
+For information on the **PowerPaste** plugin, see: xref:introduction-to-powerpaste.adoc[PowerPaste].
 
 ==== Removed translations for an error which no longer occurs.
 // TINY-10045
```

Comment:
```suggestion
==== Removed error condition, error message, and error message translations for an error which no longer occurs.
// TINY-10045

Previously, when **PowerPaste** was running with older versions of Safari as the host browser, an error condition presented, along with an associated error message, when direct image pasting was attempted.

This error condition no longer presents and, for this release, the error catching logic, associated error message, and error message translations, have all been removed.

For information on the **PowerPaste** plugin, see: xref:introduction-to-powerpaste.adoc[PowerPaste].
```
For simplicity’s and completeness’s sake, here’s a write-up of the last PowerPaste Release Notes entry. (Note, I moved the _For information…_ line to this — the last — entry in this section.